man/BSseqStat-class.Rd

```diff
@@ -0,0 +1,63 @@
+\name{BSseqStat-class}
+\Rdversion{1.1}
+\docType{class}
+\alias{BSseqStat}
+\alias{BSseqStat-class}
+\alias{[,BSseqStat-method}
+\alias{[,BSseqStat,ANY-method}
+\alias{show,BSseqStat-method}
+\title{Class BSseqStat}
+\description{
+    A class for representing statistics for smoothed whole-genome
```

Comment:
A _BSseqStat_ object can store both F-stats and t-stats, right? That's why I didn't specify the particular statistics this class represents.